Add complex Player example

Guptacos opened the pull request:
Create example that shows how to embed Spotifython into an existing
application to display playback.

Example creates a tkinter GUI, and displays current song / artist name,
along with album art. Allows user to click buttons to play, pause, skip,
and go to previous song.

Guptacos committed:
Add complex Player example

Create example that shows how to embed Spotifython into an existing
application to display playback.

Example creates a tkinter GUI, and displays current song / artist name,
along with album art. Allows user to click buttons to play, pause, skip,
and go to previous song.

spenceryu commented on examples/player_control.py:
```diff
@@ -0,0 +1,168 @@
+
+    Actual applications should get tokens using Spotify's authentication
+    pipeline, and should deal with reauthentication.
+    TODO: put reauth link
```
Should the TODOs here be fixed in a future release?

Guptacos replied:
I forgot to fix this, I'll take care of it

spenceryu commented on examples/player_control.py:
```diff
@@ -0,0 +1,168 @@
+REFRESH_RATE = 1000 # 1 second
+
+# Spotifython variables
+session = sp.Session(os.environ['TOKEN'])
```
Maybe add a comment that this is on the user to set?

Guptacos committed:
Fix complex Player example

- Change Player.next() -> Player.skip()
- Add link for more info about the Spotify auth pipeline
